### DISCUSSION TITLE
Does it run on premises?

### DISCUSSION THREAD

**Discussion by rafaelkrapf** [arc53/DocsGPT#56] [Q&A] [answered]
Can I run the tool on premises, so sensitive information is not sent online?

**Comment by dartpain** [comment]
You can run it on your own server, but unfortunately code relies too much on OpenAI's llm. But soon we will add support for others, including some that you can run localy. We already tested it and it works, just need some quality adjustments

**Comment by dartpain** [comment, marked as answer]
Recently pushed an update. Check instructions here on our [wiki](https://github.com/arc53/DocsGPT/wiki/How-to-use-different-LLM's#hosting-everything-locally) to host it all locally (on premises)
